apps/dot_ext/loggers.py

```diff
@@ -127,7 +143,12 @@ def set_session_values_from_auth_flow_uuid(request, auth_flow_uuid):
     '''
     if auth_flow_uuid:
         request.session['auth_uuid'] = str(auth_flow_uuid.auth_uuid)
-        request.session['auth_pkce_method'] = auth_flow_uuid.auth_pkce_method
+        if auth_flow_uuid.auth_pkce_method is not None:
+            request.session['auth_pkce_method'] = auth_flow_uuid.auth_pkce_method
+        if auth_flow_uuid.auth_crosswalk_action is not None:
+            request.session['auth_crosswalk_action'] = auth_flow_uuid.auth_crosswalk_action
+        if auth_flow_uuid.auth_share_demographic_scopes is not None:
+            request.session['auth_share_demographic_scopes'] = str(auth_flow_uuid.auth_share_demographic_scopes)
```

Comment:
Is there any functional difference between `if var` and `if var is not None`?

Reply:
> unctional difference between `if var` and `if var is not None`

There's a slight difference. I was originally using `if var:` and was having an issue. It can fail for other cases besides `var` not equal to `None`. This way is more specific to matching exactly the case where `var` is not equal to `None`